### ISSUE TITLE
Different masked pixel between GACOS and non GACOS

### ISSUE THREAD

**Issue by NabilAttar** [yumorishita/LiCSBAS#339]
Greeting Mr. Yumorishita

I have processed an area with GACOS and non GACOS, but they have differences in masked pixel and vel.mskd. in GACOS its too many pixels are removed so that the velocity visualization results are not good. I have tried replacing noise parameter in step 1-5 but nothing has changed in process, maybe you have suggestions for this? thank you

![1GACOS mask_ts](https://github.com/yumorishita/LiCSBAS/assets/155360570/8ddd8843-f414-4308-9eea-e1dce75b88c2)
![1non GACOS mask_ts](https://github.com/yumorishita/LiCSBAS/assets/155360570/cf2dad31-f3aa-4ee4-b9d7-3155d910b69b)
![gacos](https://github.com/yumorishita/LiCSBAS/assets/155360570/52376553-20db-499d-85dd-2e817524fb02)
![non gacos](https://github.com/yumorishita/LiCSBAS/assets/155360570/7f9054fe-56d3-470e-93ab-33294552a2b8)

**Comment by yumorishita** [comment]
The number of unwrapped images (n_unw) is much smaller in GAOCS than in non-GACOS, which could mean some of the dates have no GACOS data. Please check the output of step03.

**Comment by NabilAttar** [comment]
yes sir, in step 03, unw which originally numbered 800, became around 600. but maybe I think 600 ifg is still enough to show good mask results. Do you have any other suggestions sir? Thank You

**Comment by NabilAttar** [comment]
Thank you very much sir, its work :)
